Formula/f/flexget.rb

```diff
@@ -0,0 +1,373 @@
+class Flexget < Formula
+  include Language::Python::Virtualenv
+
+  desc "Multipurpose automation tool for content"
+  homepage "https://www.flexget.com"
+  url "https://github.com/Flexget/Flexget/releases/download/v3.17.7/flexget-3.17.7.tar.gz"
+  sha256 "f29d9b0733959341d8656d04bb49571b739c95fb327ec3b60170f83f069caa2b"
```

Comment:
Let's used the same tarball released to PyPI as that's easier for us to detect when a new stable version is released:

```suggestion
  url "https://files.pythonhosted.org/packages/b4/cc/22b27b2b61e3a3b8fb03bfb28ac43ad7713f468b470621a25c2e5161cbd1/flexget-3.17.10.tar.gz"
  sha256 "951c1c6849cac6b8a838724e317b66fecbf51d5f40c44d1e696646e0cda511b2"
```

Reply:
the latest release also changed some other dependencies, e.g: 

```
 - charset-normalizer==3.4.2
 + charset-normalizer==3.4.3
 - flexget==3.17.7
 + flexget==3.17.10
 - markdown-it-py==3.0.0
 + markdown-it-py==4.0.0
 - sqlalchemy==2.0.42
 + sqlalchemy==2.0.43
 ```
 
my question: is there some automatic way of detecting this?